deepchecks/checks/performance/class_performance_imbalance.py

```diff
@@ -0,0 +1,239 @@
+"""The class performance imbalance check."""
+import typing as t
+from functools import partial
+from collections import defaultdict
+
+import pandas as pd
+import numpy as np
+from sklearn.metrics import precision_score, recall_score, get_scorer
+from sklearn.utils.multiclass import unique_labels as get_unique_labels
+
+from deepchecks import SingleDatasetBaseCheck, Dataset, CheckResult, ConditionResult
+from deepchecks.metric_utils import task_type_validation, ModelType
+from deepchecks.string_utils import format_percent
+from deepchecks.utils import DeepchecksValueError
+
+
+__all__ = ['ClassPerformanceImbalanceCheck']
+
+
+MetricFunc = t.Callable[
+    [object, pd.DataFrame, pd.Series], # model, features, labels
+    t.Dict[t.Hashable, t.Union[float, int]] # score for each label
+]
+
+AlternativeMetric = t.Union[str, MetricFunc]
+
+
+CP = t.TypeVar('CP', bound='ClassPerformanceImbalanceCheck')
+
+
+class ClassPerformanceImbalanceCheck(SingleDatasetBaseCheck):
+    """Visualize class imbalance by displaying the difference between class metrics.
+
+    Args:
+        alternative_metrics (Mapping[str, Union[str, Callable]]):
+            An optional dictionary of metric name or scorer functions
+
+    Raises:
+        ValueError:
+            if provided dict of metrics is emtpy;
+            if one of the entries of the provided metrics dict contains name of unknown scorer;
+            if one of the entries of the provided metrics dict contains not callable value;
+    """
+
+    def __init__(
+        self,
+        alternative_metrics: t.Optional[t.Mapping[str, AlternativeMetric]] = None
+    ):
+        super().__init__()
+        self.alternative_metrics: t.Optional[t.Mapping[str, MetricFunc]] = None
+
+        if alternative_metrics is not None and len(alternative_metrics) == 0:
+            raise ValueError('alternative_metrics - expected to receive not empty dict of scorers!')
```

Comment:
raise Deepchecks error

Reply:
ok, but, I think that this error type is too overloaded with too many meanings and uses. In the python world, it is more common (appropriate) to use built-in `ValueError` to show that an incorrect value was passed to the function/

Reply:
We have a single exception type, but the idea is to indicate the exception is thrown specifically by our package and not by some dependency